bin/martinize2

```diff
@@ -33,13 +34,105 @@ from vermouth.dssp.dssp import (
     AnnotateMartiniSecondaryStructures,
     AnnotateResidues,
 )
+from vermouth.log_helpers import StyleAdapter
 from vermouth import selectors
 from vermouth.map_input import (
     read_mapping_directory,
     generate_all_self_mappings,
     combine_mappings
 )
 
+# TODO Since vermouth's __init__.py does some logging (KDTree), this may or may
+# not work as intended. Investigation required.
+
+LOGGER = logging.getLogger('vermouth')
+
+
+class BipolarFormatter:
```

Comment:
I put these here because of the separation front-end <=> library. The LoggerAdapters are part of the library since you need (can use) them to make logging calls. The Formatter and Handler are here, because the *library* doesn't need/use them. Now, if we were to make a martinize2 module...

I'll give it some more thought.

Reply:
It is indeed kind of middle ground between vermouth and martinize. Ideally, we want to empty martinize itself from as much code as possible, especially code like the logger that can easily be used by an other vermouth-based tool. We could have a `interface.py` module.